### PR TITLE
1065 updated styles for create page

### LINKED ISSUE #1065: Create contribution page UI changes
Update UI for create contribution page (form) as per Zeplin design.
Change the font styles, buttons, input fields.
Include pop-ups UI changes as well. 

Get design elements for
- Design for pop-up screens
- Eye icon, radio button, close icon, form fields

### PR DESCRIPTION
**Pull request by minump**
## Description
Create contribution page styling updated as per zeplin design. The /create and /edit pages are updated in this PR.
Updated styles : 
input fields, checkboxes, buttons and pop-ups (cancel and submit pop-ups).

Awaiting designer input for : 
close button (close icon), eye icon. 

**Fixes #1065 [#1070](https://github.com/rokwire/rokwire-building-blocks-api/issues/1070)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: contributions/catalog/webapps/static/css/style.css [hunk 1]
```diff
@@ -78,24 +79,94 @@ html * {
...
+    font-weight: 500;
+    font-stretch: normal;
+    font-style: normal;
+    letter-spacing: normal;
+    height: 20px;
+    color: #13294b;
+}
+
+input[type=text], input[type=url], select, textarea {
+    color: #404040;
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/static/css/style.css:93]:
```
.Rectangle-Copy-6 {
  width: 288px;
  height: 48px;
  padding: 12px 8px 16px;
  border: solid 1px #002855;
  background-color: #fff;
}

input {
  width: 272px;
  height: 20px;
  font-family: ProximaNova;
  font-size: 16px;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: 1.25;
  letter-spacing: normal;
  color: #404040;
}
```

Copying the textbox style and hint text from Zeplin. 

Please remove the rounded edges currently present, and fix the height and style of the hint text style (if possible).

File: contributions/catalog/webapps/static/css/style.css [hunk 2]
```diff
@@ -78,24 +79,94 @@ html * {
...
-    padding: 12px;
-    border: 1px solid #ccc;
+    padding: 5px;
+    border: 1px solid #002855;
+    resize: vertical;
+    background-color: #fff;
     border-radius: 4px;
+}
+
+input[type=number], input[type=date]{
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/static/css/style.css:102]: Is there a difference in the styles of different inputs like numbers, dates, etc.? Do we need separate style classes for that?

**Comment by sandeep-ps** [comment]
Visually, the other changes look good to me. I had a question about the label text style, which I have added to the meeting notes page where we are collating all design questions. This is coming out really well! Thanks.

**Comment by sandeep-ps** [comment]
@bingzhang, Please go through this PR and share your feedback when you get a chance. Thanks.

File: contributions/catalog/webapps/static/css/style.css (hunk 2 above)
Comment (minump) [reply contributions/catalog/webapps/static/css/style.css:102]: input type text , url and textarea have width 100%. input type numbers and dates have other specified widths.

File: contributions/catalog/webapps/static/css/style.css (hunk 1 above)
Comment (minump) [reply contributions/catalog/webapps/static/css/style.css]: updated box styling and placeholder fonts

**Comment by minump** [comment]
Updated styles to match zeplin design.
Ready for review @sandeep-ps @bingzhang

File: contributions/catalog/webapps/static/css/style.css (hunk 2 above)
Comment (sandeep-ps) [reply contributions/catalog/webapps/static/css/style.css]: Thanks for clarifying, Minu.

File: contributions/catalog/webapps/static/css/style.css (hunk 1 above)
Comment (sandeep-ps) [reply contributions/catalog/webapps/static/css/style.css]: Thanks.

**Review by sandeep-ps — APPROVED** [review]
I am approving this PR. There may be some further fine-tuning that can be worked on later. Thanks.

**Comment by minump** [comment]
Merging this PR.

**Pull request merged**